solvers/stereo_imaging/time_window_pruned_stereo_milp/tests/test_stereo_imaging_time_window_pruned_stereo_milp.py

```
@@ -0,0 +1,1533 @@
+        not (Path(__file__).resolve().parents[2] / "benchmarks" / "stereo_imaging" / "dataset" / "cases" / "test" / "case_0001").exists(),
+        reason="case_0001 dataset not present",
+    )
+    def test_three_runs_identical(self, tmp_path):
+        import subprocess
+
+        repo_root = Path(__file__).resolve().parents[2]
+        solver_dir = repo_root / "solvers" / "stereo_imaging" / "time_window_pruned_stereo_milp"
+        case_dir = repo_root / "benchmarks" / "stereo_imaging" / "dataset" / "cases" / "test" / "case_0001"
+        config_dir = solver_dir / "config.example.yaml"
+
+        statuses: list[dict[str, Any]] = []
```

Ruff (0.15.11)

[error] Undefined name `Any`

(F821)